src/sage/topology/simplicial_complex.py

```diff
@@ -1204,6 +1274,39 @@ def __ne__(self, right):
         """
         return not self.__eq__(right)
 
+    def __le__(self, right):
+        """
+        Return ``True`` iff every face of ``self`` is a face of ``right``.
+
+        EXAMPLES::
+
+            sage: SimplicialComplex([1]) <= SimplicialComplex([[0,1], [1,2]])
+            True
+            sage: SimplicialComplex([[1,3]]) <= SimplicialComplex([[0,1], [1,2]])
+            False
+        """
+        faces = right.faces()
+        for facet in self._facets:
+            dim = facet.dimension()
+            if dim not in faces:
+                return False
+            if facet not in faces[dim]:
+                return False
+        return True
```

Comment:
```suggestion
        return all(facet in faces.get(facet.dimension(), []) for facet in self._facets)
```
Here is a 1-line version.